src/main/java/org/humancellatlas/ingest/process/ProcessEventHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.humancellatlas.ingest.process;
+
+import lombok.AllArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.humancellatlas.ingest.core.service.ValidationStateChangeService;
+import org.humancellatlas.ingest.protocol.Protocol;
+import org.humancellatlas.ingest.state.ValidationState;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.rest.core.annotation.*;
+
+import java.util.Set;
+
+import static org.humancellatlas.ingest.core.EntityType.PROCESS;
+
+@RepositoryEventHandler()
+@RequiredArgsConstructor
+public class ProcessEventHandler {
+
+    private final @NonNull ValidationStateChangeService validationStateChangeService;
+
+    @HandleAfterLinkDelete
+    public void handleProtocolLinkingAfterDelete(Process process, Set<Protocol> protocols){
+        validationStateChangeService.changeValidationState(PROCESS, process.getId(), ValidationState.DRAFT);
```

Comment:
I would actually prefer if you set the graph validation state directly to Pending whenever linking changes. Setting the metadata back to draft will trigger the metadata validation which isn't necessary when you're not changing the content. Likewise, we also don't trigger graph validation when only the content has changed. This can be an optimisation that can be done later. Let's see and test what happens.